tests/unit/type/string.tcl

```diff
@@ -102,6 +102,75 @@ start_server {tags {"string"}} {
         assert_equal 20 [r get x]
     }
 
+    test "GETEX EX option" {
+        r del foo
+        r set foo bar
+        r getex foo ex 10
+        set ttl [r ttl foo]
+        assert {$ttl <= 10 && $ttl > 5}
+    }
+
+    test "GETEX PX option" {
+        r del foo
+        r set foo bar
+        r getex foo px 10000
+        set pttl [r pttl foo]
+        assert {$pttl <= 10000 && $pttl > 5000}
+    }
+
+    test "GETEX EXAT option" {
+        r del foo
+        r set foo bar
+        r getex foo exat [expr [clock seconds] + 10]
+        set ttl [r ttl foo]
+        assert {$ttl <= 10 && $ttl > 5}
+    }
+
+    test "GETEX PXAT option" {
+        r del foo
+        r set foo bar
+        r getex foo pxat [expr [clock milliseconds] + 10000]
+        set pttl [r pttl foo]
+        assert {$pttl <= 10000 && $pttl > 5000}
+    }
+
+    test "GETEX DEL option" {
+        r del foo
+        r set foo bar
+        assert_equal bar [r getex foo del]
+        assert_equal {} [r getex foo del]
+    }
+
+    test "GETEX PERSIST option" {
+        r del foo
+        r set foo bar ex 10
+        set ttl [r ttl foo]
+        assert {$ttl <= 10 && $ttl > 5}
+        r getex foo persist
+        assert_equal -1 [r ttl foo]
+    }
+
+    test "GETEX KEEPTTL option" {
+        r del foo
+        r set foo bar ex 10
+        set ttl [r ttl foo]
+        assert {$ttl <= 10 && $ttl > 5}
+        r getex foo keepttl
+        assert {$ttl <= 10 && $ttl > 5}
+    }
+
+    test "GETEX syntax errors" {
+        set ex {}
+        catch {r getex foo non-existant-option} ex
+        set ex
+    } {*syntax*}
+
+    test "GETEX no optional argument" {
+         set ex {}
+         catch {r getex foo} ex
```

Comment:
i thought the docs next to the command said KEEPTTL is the default (which i think it should be, i.e. making it behave like GET by default).
i think the arity of the command should be changed and this test altered.

Reply:
Fixed and changed arity.